scripts/new.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Dirty 🤤🤤🤤

I even have template user snippets for each one I'll beef this up a bit

Reply:
💯